assets/js/blocks/products/all-products/index.js

```diff
@@ -74,4 +73,37 @@ registerBlockType( 'woocommerce/all-products', {
 			</div>
 		);
 	},
+};
+
+/**
+ * Register and run the "All Products" block.
+ */
+registerBlockType( 'woocommerce/all-products', {
+	...blockSettings,
+	/**
+	 * Deprecation rule to handle the previous default rows which was 1 instead of 3.
+	 */
+	deprecated: [
+		{
+			attributes: Object.assign( blockSettings.attributes, {
+				rows: {
+					type: 'number',
+					default: 1,
+				},
+			} ),
```

Comment:
I had to revert this. Looks like it doesn't actually affect things. This did however change the original object which caused the default to always be 1. Fixed in latest push :)

Reply:
Oh, it's because of improper syntax that I missed.  If you do this instead it should be fine:

```js
attributes: Object.assign( {}, blockSettings.attributes, rows: { type: 'number', default: 1, } ),
```

Reply:
Hmm I'm definitely getting some type of reordering bug with the data-attributes. I'm not sure why yet.